Code/skyrim_ui/src/assets/i18n/nl.json

```diff
@@ -0,0 +1,149 @@
+{
+  "INFO": "DO NOT MODIFY THIS FILE! USE THE overwrite.json FILE TO TRANSLATE THE UI!",
+
+  "COMPONENT": {
+    "CHAT": {
+      "SEND": "Verzenden",
+      "MESSAGE": "Bericht",
+      "MESSAGE_TOO_LONG": "Je kan geen bericht langer dan {{chatMessageLengthLimit}} verzenden."
+    },
+    "CONNECT": {
+      "INFO": {
+        "SERVER_ADDRESS": "Verbind met een server door het volledige adres te gebruiken. Gebruik het formaat <strong>adres:poort</strong>. De standaard poort is <strong>10578</strong>.",
+        "HELGEN": "Verbind niet met een server wanneer je nog in de Helgen introductie bent. Zorg ervoor dat je eerst uit Helgen vlucht."
+      },
+      "ADDRESS": "Adres",
+      "PASSWORD": "Wachtworod",
+      "SAVE_PASSWORD": "Wachtwoord opslaan?",
+      "CONNECT": "Verbinden",
+      "CANCEL": "Annuleren",
+      "PUBLIC_SERVERS": "Openbare servers",
+      "ERROR": {
+        "CONNECTION": "Kan niet verbinden met de desbetreffende server. Wees er zeker van dat je het juiste adres hebt ingevuld en dat je geen netwerkproblemen ondervind.",
+        "VERSION_MISMATCH": "Kan niet verbinden met de desbetreffende server omdat het niet dezelfde versie heeft van Skyrim Together als jij.",
+        "INVALID_ADDRESS": "Het adres is in een ongeldig formaat."
+      }
+    },
+    "DISCONNECT": {
+      "PROMPT": "Weet je het zeker dat je de verbinding wil verbreken?",
+      "PROCEED": "Doorgaan",
+      "CANCEL": "Annuleren",
+      "ERROR": {
+        "CONNECTION": "Kan niet verbinden met de desbetreffende server. Wees er zeker van dat je het juiste adres hebt ingevuld en dat je geen netwerkproblemen ondervind.",
+        "VERSION_MISMATCH": "Kan niet verbinden met de desbetreffende server omdat het niet dezelfde versie heeft van Skyrim Together als jij.",
+        "INVALID_ADDRESS": "Het adres is in een ongeldig formaat."
+      }
+    },
+    "ERROR": {
+      "OK": "Oke"
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Bericht van"
+    },
+    "PARTY_MENU": {
+      "ACCEPT_INVITE": "Accepteer uitnodiging van {{name}}",
+      "LAUNCH_PARTY" : "Maak groep",
+      "EXPERIENCE_WARNING": "Voor een optimale 'questing' ervaring, lees de the speelgids op onze wiki, welke je kan vinden op onze website, <b>skyrim-together.com</b>. Hier is een korte opsomming van de richtlijnen:",
+      "GUIDELINES": {
+        "PARTY": "Door het maken van een groep, wordt jij een soort van  \"host\", aka de groepsleider.",
+        "PARTY_LEADER_TALK": "Alleen de groepsleider kan praten tegen quest NPCs om de quests voort te zetten.",
+        "PARTY_LEADER_LOOT_QUESTITEMS": "Alleen de groepsleider kan quests items looten.",
```

Comment:
quests -> quest
Also maybe the last few words need to be translated a bit more, something like "quest voorwerpen oprapen"?